.github/workflows/build.yml

```diff
@@ -35,20 +35,23 @@ jobs:
   e2e:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: engineerd/setup-kind@v0.3.0
-    - uses: engineerd/configurator@v0.0.1
-      with:
-        name: just
-        url: https://github.com/casey/just/releases/download/v0.5.8/just-v0.5.8-x86_64-unknown-linux-musl.tar.gz
-        pathInArchive: just
-    - name: Run e2e tests
-      run: |
-        cd crates/wascc-provider
-        cargo build --bin krustlet-wascc --manifest-path ../../Cargo.toml
-        ../../target/debug/krustlet-wascc --node-name krustlet-wascc --node-ip 172.17.0.1 --port 3000 &
-        cd ../wasi-provider
-        cargo build --bin krustlet-wasi --manifest-path ../../Cargo.toml
-        ../../target/debug/krustlet-wasi --node-name krustlet-wasi --node-ip 172.17.0.1 --port 3001 &
-        cd ../..
-        just test-e2e
+      - uses: actions/checkout@v2
+      - uses: engineerd/setup-kind@v0.3.0
+      - uses: engineerd/configurator@v0.0.1
+        with:
+          name: just
+          url: https://github.com/casey/just/releases/download/v0.5.8/just-v0.5.8-x86_64-unknown-linux-musl.tar.gz
+          pathInArchive: just
+      - name: Run e2e tests
+        env:
+          KRUSTLET_NODE_IP: "172.17.0.1"
+          # The default location for the cert is not accessible on build hosts, so do it in the current dir
+          KEY_DIR: "./config"
+          PFX_PATH: "./config/certificate.pfx"
+        run: |
+          ifconfig
+          nslookup `hostname`
```

Comment:
Can we add a comment to the `ifconfig/nslookup` magic

Reply:
This was just debug material while I am getting the e2e tests working